types/react-router-redux/tslint.json

```diff
@@ -1 +1,6 @@
-{ "extends": "../tslint.json" }
+{
+    "extends": "../tslint.json",
+    "rules": {
+        "forbidden-types": false
```

Comment:
You shouldn't need this options. You're not using any forbidden types such as `Function`, `Object` etc.